extensions/amazon-dynamodb/runtime/src/main/java/io/quarkus/dynamodb/runtime/SyncHttpClientConfig.java

```diff
@@ -0,0 +1,211 @@
+package io.quarkus.dynamodb.runtime;
+
+import java.net.URI;
+import java.time.Duration;
+import java.util.List;
+import java.util.Optional;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+
+@ConfigGroup
+public class SyncHttpClientConfig {
+
+    /**
+     * Type of the sync HTTP client implementation
```

Comment:
You should mention the accepted values here (APACHE, URL)

Reply:
Added